test/test_nn.py

```diff
@@ -6844,9 +6844,6 @@ def test_gaussian_nll_loss_args(self):
             target = torch.randn(3, 5)
             var = torch.ones(3, 3)
             torch.nn.functional.gaussian_nll_loss(input, target, var)
-        with self.assertRaisesRegex(ValueError, 'var has negative entry/entries'):
```

Comment:
I think the previous assertion can also be an ErrorInput and this test removed?

Reply:
While I suspect this test can be removed entirely, `test_gaussian_nll_loss_broadcasting` and `test_gaussian_nll_loss_reduction_modes` may need to stick around for now -- but we should add a comment that they should eventually be made redundant with the OpInfo